app/models/pawn.rb

```diff
@@ -1,2 +1,67 @@
 class Pawn < Piece
+
+  def valid_move?(destination_x, destination_y)
+    #invalid move error raises if moving to same current
+    #xy coordinates
+
+    #check if it's backward/horizontal move
+    invalid_move = invalid_move?(destination_y)
+    return false if invalid_move
```

Comment:
instead of `invalid_move?`. I'd rename the method to `backward_or_horizontal_move?`

Then you can do

`return false if backward_or_horizontal_move?`

And you can delete the comment, because the code is very readable :)